src/Twilio/Clients/TwilioRestClient.cs

```diff
@@ -146,7 +146,7 @@ private static Response ProcessResponse(Response response)
                 throw new ApiConnectionException("Connection Error: No response received.");
             }
 
-            if (response.StatusCode >= HttpStatusCode.OK && response.StatusCode < HttpStatusCode.Ambiguous)
+            if (response.StatusCode >= HttpStatusCode.OK && response.StatusCode < HttpStatusCode.BadRequest)
```

Comment:
Why this change? Won't this now throw an exception for the 400 status codes?

Reply:
`HttpStatusCode.BadRequest` is `400`. So this will return the response if the code is in [200,400).

Reply:
Ah, for some reason my google search was telling me HttpStatusCode.Ambiguous was 502, not 300.